server/API/UploadEnrollements.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+include_once '../functions.php';
+
+define("CODE", 'Code');
+define("STUDENT", 'Student');
+
+if (!function_exists("upload_enrollments")) {
+
+	function upload_enrollments($vars) {
+
+		try {
+			$userId = intval($_SESSION['user_id']);
+			$pdo = new PDO("sqlite:../db.sqlite");
+
+			// Get type of user
+			$stmt = $pdo->prepare('SELECT * FROM users WHERE ID = :userId');
+			$stmt->bindValue(':userId', $userId, PDO::PARAM_INT);
+
+			if (!$stmt->execute()) {
+				throw new PDOException($stmt->errorInfo()[2]);
+			}
+			$userData = $stmt->fetch();
```

Comment:
if you cherry pick (git cherry-pick commit-id) my commit with get_myself(), you can use that to get the logged_user

Reply:
The commit you are referring to (e09dac243ef4415ffe9383c4f898566a59decd09) contains other changes other than the function get_myself. I would prefer to not commit those changes in this branch, as they not belong here.